src/rest/SearchEndpoint.ts

```diff
@@ -382,6 +383,42 @@ export class SearchEndpoint implements ISearchEndpoint {
       return results;
     });
   }
+
+  /**
+   * Performs a search on the index and returns a Promise of [`IPlanResults`]{@link IPlanResults}.
```

Comment:
Trying to explain the meaning of /plan. You basically get info about the query but not the results.